### DISCUSSION TITLE
Rechenfehler oder Denkfehler

### DISCUSSION THREAD

**Discussion by mar1ged** [tbnobody/OpenDTU#394] [Q&A] [answered]
Ich hatte mir heute morgen einen Screenshot erstellt, zu der Zeit war es knapp eine Viertelstunde her dass der WR begann Strom zu produzieren.
Wenn ich mir abends die Gesamtproduktion anschaue, kann das irgendwie nicht hinhauen. Der Wert ist fast doppelt so hoch.

7:53 15,32 kWh total yield total
165 Wh total yield day

19:33 17,24 kWh total yield total
1.068 Wh total yield day

Rechnet hier irgendwas falsch oder mache ich einen Denkfehler ? Aktuell nutze ich 8ba0a65

**Comment by tbnobody** [comment]
Ja, das ist mir beim ersten analysieren der Rohdaten schon aufgefallen. Die Werte kommen so aus dem Inverter...

**Comment by mar1ged** [comment]
Ok, Danke für die Info. Dann ist das kein Fehler, an dem OpenDtu etwas ändern kann.
Weiß man ggf. ob der Gesamtwert stimmt ?

**Comment by tbnobody** [comment]
Ich sag mal so, ich habe einen Inverter Port an dem kein Panel angeschlossen ist. Die Anschlüsse sind einfach unbelegt. Jetzt nach einem guten Jahr sagt YieldTotal für diesen Kanal 2,3 kWh.... Ich gehe mal davon aus das eine gewisse Diskrepanz immer vorhanden ist.

**Reply by mar1ged** [reply]
Ok, das ist ja witzig. Weißt Du zufällig, ob jemand Mal die Werte des WR mit denen eines Shelly verglichen hat ?
Wenn der Hoymiles nicht 100% richtig misst ist das eine Sache. Aber dass zwei Werte, bei denen einer eigentlich nur den anderen aufsummiert, sich so widersprechen, ist schon schräg.

**Comment by Solarteur** [comment, marked as answer]
![ahoy_DTU_AC_DC](https://user-images.githubusercontent.com/117977338/204226825-210ca387-69df-4248-8974-ebb766ae9bfb.png)
![OpenDTU_AC_DC](https://user-images.githubusercontent.com/117977338/204227551-3eee0eab-4d89-4ad8-b8cd-76a067070367.png)
Vergleich mit Shelly EM.
Bei dem Beispiel mit der ahoy-DTU ist zusätzlich noch der Momentanwert AC in den Vergleich einbezogen. 
Beide Beispiele zeigen aus meiner Sicht, dass der oben erwähnte "Tagesfehler" nicht kumuliert wird.

Zur Messgenauigkeit der Wechselrichter im Allgemeinen: Die Elektrizitätszähler des Messstellenbetreibers haben je nach Bauart eine Eichfehlergrenze von +/- 2 bis 3%. Eingebaut kommt die Verkehrsfehlergrenze zum Tragen - die beträgt das Doppelte der Eichfehlergrenze (Quelle: https://www.nzr.de/home/loesungen/elektrizitaetszaehler/allgemeines-zu-elektrizitaetszaehlern/)

**Comment by MrReSc** [comment]
Ich habe alle Tageswerte in eine Datenbank abgespeichert. Die Summe aus diesen "yield day" Werten ergibt exakt den "yield total" Wert vom Wechselrichter. Bei mir stimmen die Messwerte.

**Reply by mar1ged** [reply]
Vielen Dank an alle für die Infos

**Comment by homeautomation2022** [comment]
Ob die nun AC oder DC sind, bin ich mir nicht ganz sicher, aber YieldTotal ist von uns erfunden und einfach nur die Summer der Einzelkanäle

**Reply by MrReSc** [reply]
Als wenn es DC ist, müsste man noch die den Wirkungsgrad mit einrechnen.
